lib/krb5/send_to_kdc.c

```diff
@@ -393,6 +394,8 @@ deallocate_host(void *ptr)
     if (!rk_IS_BAD_SOCKET(host->fd))
 	rk_closesocket(host->fd);
     krb5_data_free(&host->data);
+    free(host->freeai);
```

Comment:
Shouldn't this be `freeaddrinfo()` instead of `free()`?  Should this be contingent on `host->freeai` not being `NULL`?